cluster/config-defaults.yaml

```diff
@@ -401,11 +407,22 @@ external_dns_ownership_prefix: ""
 # domains that should be ignored by ExternalDNS
 external_dns_excluded_domains: cluster.local
 
+# select which cache to use for Cluster DNS
+{{ if eq .Cluster.Environment "production" }}
+dns_cache: "dnsmasq"
+{{ else }}
+dns_cache: "unbound"
+{{ end }}
+
 # DNS container resources
 dns_dnsmasq_cpu: "100m"
 dns_dnsmasq_mem: "50Mi"
 dns_dnsmasq_sidecar_cpu: "10m"
 dns_dnsmasq_sidecar_mem: "45Mi"
+dns_unbound_cpu: "100m"
```

Comment:
I think it should be fairly safe to give it a bit more memory, especially at the beginning. Keeping the values unchanged from dnsmasq (or at least their totals) would also make it easier to switch back and forth in case we need to rollback.

Reply:
Changed to use the same settings as for dnsmasq.